backend/cfg/cfg_ls_utils.mli

```diff
@@ -0,0 +1,108 @@
+[@@@ocaml.warning "+a-4-30-40-41-42"]
+
+open Cfg_regalloc_utils
+
+val ls_debug : bool
+
+val ls_verbose : bool Lazy.t
+
+val ls_invariants : bool Lazy.t
+
+val log :
+  indent:int -> ?no_eol:unit -> ('a, Format.formatter, unit) format -> 'a
+
+val log_body_and_terminator :
+  indent:int ->
+  Cfg.BasicInstructionList.t ->
+  Cfg.terminator Cfg.instruction ->
+  liveness ->
+  unit
+
+module Order : sig
+  type t =
+    | Layout (* Order given by iterating over the CFG layout. *)
+    | DFS (* Order computed by a depth-first search from the entry point. *)
+
+  val all : t list
+
+  val to_string : t -> string
+
+  val value : t Lazy.t
+end
+
+val iter_cfg_order :
+  Cfg_with_layout.t -> Order.t -> f:(Cfg.basic_block -> unit) -> unit
+
+val iter_instructions_order :
+  Cfg_with_layout.t ->
+  Order.t ->
+  instruction:(trap_handler:bool -> Cfg.basic Cfg.instruction -> unit) ->
+  terminator:(trap_handler:bool -> Cfg.terminator Cfg.instruction -> unit) ->
+  unit
```

Comment:
Please add a comment about the meaning of `trap_handler` parameter: true for the first instruction in a block that is marked as a trap_handler, right?

Reply:
Added a comment.